src/borg/archiver.py

```diff
@@ -1252,8 +1259,12 @@ def _delete_repository(self, args, repository):
                 if not keep_security_info:
                     SecurityManager.destroy(repository)
             else:
-                logger.info("Would delete repository.")
-                logger.info("Would %s security info." % ("keep" if keep_security_info else "delete"))
+                if self.output_list:
+                    logging.getLogger('borg.output.list').info("Would delete repository.")
+                    logging.getLogger('borg.output.list').info("Would %s security info." % ("keep" if keep_security_info else "delete"))
+                else:
+                    logger.info("Would delete repository.")
+                    logger.info("Would %s security info." % ("keep" if keep_security_info else "delete"))
```

Comment:
guess this is a bit questionable regarding whether this counts as "list".

as borg always operates on 1 repo, generic hints about this repo and the related cache are not "listing" anything.

only when it outputs all archives in that repo, that would be "listing".

Reply:
Hum, then I don't know which logger to use.
If I use the info level, there is no difference between the --list and nothing for the repositories (which can be a consistent way of acting). To be sure that something is written I could use the warn level.

Reply:
@ThomasWaldmann What's your idea on this?
Should I use the warn level to be sure to print something?

Reply:
this is not a list, so no change is needed. giving --list or not should not make a difference here.

1244-1247 is the "listy" stuff here:
- with --list: like now
- without --list: just tell that "N archives are going to get deleted (use --list for details)"